xml/System.Net/WebUtility.xml

```diff
@@ -424,6 +424,9 @@
   
 -   `(` and `)` (opening and closing parentheses)  
   
+ Each ' ' (space) character is converted to a `+` (plus) character.
+ For web applications, the <xref:System.Web.HttpUtility.UrlDecode%2A> method can be used instead, and produces identical results.
```

Comment:
This line should be removed. While the statement is true, we would rather developers use the System.Net.WebUtility class. We don't want to encourage more use of System.Web.HttpUtility class.

System.Net.WebUtility was added several years ago when we were developing .NET Core. We were trying to remove the use of System.Web in generally since it was part of the ASP.NET code that targets .NET Framework and linking with that assembly brings in a lot of unrelated code. ASP.NET Core, on the other hand, doesn't bring in the System.Web namespace.

We are aware that System.Net.WebUtility is "duplicative" of the original System.Web.HttpUtility class. In fact, some developers have pointed this out. See: https://github.com/dotnet/corefx/issues/20795.